HW03_Graphs.Rmd

```diff
@@ -0,0 +1,174 @@
+---
+title: "HW03_Graphs"
+author: "Ricardo Muñiz Trejo"
+date: "20/7/2020"
+output: github_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+```{r load libraries and set format, echo = FALSE}
+library("readr")
+library("readxl")
+library("ggplot2")
+theme_set(theme_bw())
+```
+
+
+## Introduction
+
+For this assignment, I decided to recreate some of the graphs of my master's thesis. Originally, my graphs were created with Origin 7 (images included for comparison). Because this is unpublished data, I censored the names, but basically the data presented here is from the physical-chemical characterization a couple of thermophilic enzymes.
+
+These aren't exactly my original graphs since my thesis is in Spanish, so I needed to convert them to English first. Also, my recreated graphs aren't meant to be perfect. I find that ggplot2 is much more versatile than Origin in customizing graphs and I like that a lot :heart:. 
+
+To practice importing files with different R packages, I decided to divide this homework in three sections: 
+
+- utils 
+- readr 
+- readxl
+
+## utils package
+
+For this section, I'm going to recreate circular dichroism (*CD*) and catalytic activities graphs, for that purpose, I created a couple of *txt* files (available in data), one with data separated by slashes "/", and the other with tabulate-separated values.
+
+#### Circular Dichroism
+
+Let's start with CD spectra.
+
+This one is the original graph:
+![Circular Dicroism](images/dicro.JPG)
+
+And here's the code for its recreation using ggplot2:
+
+```{r CD, echo = TRUE}
+
+#I choose some colors and shapes for each species beforehand from a colorblind-friendly palette. To simplify the scripts, here are some vectors for it
+colorg <- c("#E69F00","#D55E00")
+shapeg <- c(25, 23)
+
+dc_data <- read.table("data/dichroism.txt", 
+                      header = TRUE, 
+                      sep = "/", 
+                      stringsAsFactors = FALSE)
+
+#CD data, Mean Residue Ellipticity ([theta]), is usually divided by 1000 to facilitate visualization
+dc_data$Ellip_mod <- dc_data$Ellipticity / 1000
+
+ggplot(data = dc_data, mapping = aes(x = Wavelength, y = Ellip_mod, color = Species, shape = Species)) +
```

Comment:
Since the graph has so much white space in the upper right-hand corner, it could be a better use of space to move your legend inside the graph like on your original graph.